stdlib/public/core/OutputStream.swift

```diff
@@ -330,11 +330,14 @@ internal struct _Stdout : OutputStreamType {
   }
 
   mutating func write(string: String) {
-    // FIXME: buffering?
-    // It is important that we use stdio routines in order to correctly
-    // interoperate with stdio buffering.
-    for c in string.utf8 {
-      _swift_stdlib_putchar_unlocked(Int32(c))
+    let utf8 = string.nulTerminatedUTF8
```

Comment:
Hmm, thanks for your feedback.  Is there an alternative?  We don't want the null termination, just the string bytes.